### PR TITLE
bears/python: Add PyFlakesBear

### PR DESCRIPTION
**Pull request by srisankethu**
Closes https://github.com/coala/coala-bears/issues/825

### CODE REVIEW

**Comment by aptrishu** [comment]
why is the coverage not 100% ? Its lacking coverage in other modules. :(

**Comment by Mixih** [comment]
no, it just setuptools being stupid. Rebase and it should go away....

**Comment by Mixih** [comment]
@srisankethu I know you know this, but it's a bad idea to develop on master, even for little things. In the future, always make branches.

**Comment by Mixih** [comment]
@srisankethu Please rebase against master again.

**Comment by Mixih** [comment]
@aptrishu I think we only care about the circle coverage, the windows one is always missing things because of skips

**Comment by srisankethu** [comment]
@Mixih Cool :+1:

File: requirements.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ pyyaml~=3.12
 vulture~=0.10.0
 bandit~=1.2
 nbformat>=4.1
+pyflakes~=1.2.0
```
Comment (jayvdb) [inline requirements.txt]: Why isnt this 1.3 ; that has been requested a few times.

File: tests/python/PyFlakesBearTest.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+"""
+
+bad_file = """
+import os
+print("Hi")
+"""
+
+PyFlakesBear = verify_local_bear(PyFlakesBear,
+                                 valid_files=(good_file,),
+                                 invalid_files=(bad_file,))
```
Comment (jayvdb) [inline tests/python/PyFlakesBearTest.py]: `bad_file` should only be level info messages; does `verify_local_bear` consider level info messages to be an invalid file?

File: requirements.txt (hunk 1 above)
Comment (srisankethu) [inline requirements.txt]: @jayvdb I changed it to 1.3 earlier but it seems to be failing again with same flake8 requirement error.

Comment (srisankethu) [inline requirements.txt]: https://circleci.com/gh/coala/coala-bears/4151?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: tests/python/PyFlakesBearTest.py (hunk 2 above)
Comment (adtac) [inline tests/python/PyFlakesBearTest.py]: @srisankethu [yes](https://github.com/coala/coala/blob/master/coalib/testing/LocalBearTestHelper.py#L74)

File: requirements.txt (hunk 1 above)
Comment (jayvdb) [inline requirements.txt]: OK. Now lets investigate options on how to fix that. 1.2 is *not* an acceptable option.

Comment (jayvdb) [inline requirements.txt]: I believe I suggested last time to try uninstalling flake8.  Did you try that?

**Comment by jayvdb** [comment]
Turns out, it was because `flake8` was cached.  Doing a build without a cache ... green.

**Comment by jayvdb** [comment]
ack 7739e93

**Comment by jayvdb** [comment]
@rultor merge

**Comment by jayvdb** [comment]
Needs a rebase?

**Comment by srisankethu** [comment]
@jayvdb Done :+1:

**Comment by Mixih** [comment]
ack 25c9283

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**